### PR TITLE
Java wechaty transplant midpoc

### PR DESCRIPTION
**Pull request by LuSingTea**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.

``` md
---
title: "'Score Your Face Photo' a ML&Wechaty practice"
author: huyingxi
image: /assets/2020/your-blog-title/teaser.jpg
---

YOUR_FIRST_SENTENCE_SHOULD_BE_A_SUMMARY_FOR_YOUR_POST

```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/assets/2020` directory.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.js.org/blob/master/_developers/YOUR_GITHUB_USERNAME.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.js.org#contribute-guideline)

### CODE REVIEW

File: _posts/2020-08-17-java-wechaty-transplant-midpoc.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+## 暑期2020 [Wechaty Java组件移植] POC 成果展示
+
+| title                     | date                   | author       | categories | image                                                        | tags         |
```
Comment (lijiarui) [inline _posts/2020-08-17-java-wechaty-transplant-midpoc.md]: Please add the blog as the template show, and then I can review your blog.

File: _data/authors.yml [hunk 2]
```diff
@@ -971,3 +971,14 @@ csjuXYZ:
     - label: "Email"
       icon: "fas fa-fw fa-envelope-square"
       url: "hoc0415@foxmail.com"
+cunkoulaocai:
+  name        : "陈炀"
+  bio         : "java开发,一个小白"
+  avatar      : "https://avatars3.githubusercontent.com/u/30389241?s=460&u=99b73e1a68a668a56e0a567ffa33bb8c2e99b4bb&v=4"
```
Comment (lijiarui) [inline _data/authors.yml:977]: Please change this author.yml to a developer file.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

File: _data/authors.yml (hunk 2 above)
Comment (lijiarui) [reply _data/authors.yml:977]: ping

File: _posts/2020-08-17-java-wechaty-transplant-midpoc.md (hunk 1 above)
Comment (lijiarui) [reply _posts/2020-08-17-java-wechaty-transplant-midpoc.md]: ping

**Review by lijiarui — CHANGES_REQUESTED** [review]
ping

**Comment by lijiarui** [comment]
ping @cunkoulaocai

**Comment by LuSingTea** [comment]
pong

**Comment by lijiarui** [comment]
Could you change as I request as soon as possible? Since other soc contributors already posted their blog for a few weeks but you haven't posted even one blog!

I will close the PR if this PR has no new commit at the end of September.

**Comment by LuSingTea** [comment]
我这边为什么渲染不出对应的格式
image
image
我格式应该是正确的了

**Pull request closed without merging**